fppgen/lib/autograde.py

```diff
@@ -71,7 +72,7 @@ def info_json_update(self) -> Dict[str, Union[str, Dict[str, Union[bool, str, in
         }
 
     def populate_tests_dir(self, test_dir: str, answer_code: str, setup_code: str, test_region: str,
-                    pre_code: str='', post_code: str='', log_details: bool= True) -> None:
+                    pre_code: str = '', post_code: str = '', log_details: bool = True) -> None:
```

Comment:
at some point we should settle on a code formatter

Reply:
I totally forgot to add prettier to the devcontainer + pre-commit hooks, will do that in another PR